cpp/server/handler.cc

```diff
@@ -289,10 +289,13 @@ void HttpHandler::Add(libevent::HttpServer* server) {
   // that they should be spun off to the thread pool.
   AddProxyWrappedHandler(server, "/ct/v1/get-entries",
                          bind(&HttpHandler::GetEntries, this, _1));
-  // Don't really need to proxy this one, but may as well just to keep
-  // everything tidy:
-  AddProxyWrappedHandler(server, "/ct/v1/get-roots",
-                         bind(&HttpHandler::GetRoots, this, _1));
+  // TODO(alcutter): Support this for mirrors too
+  if (cert_checker_) {
```

Comment:
The changes here are all related to mirroring, right? Because this is currently always true, if I'm not mistaken?